src/ssv-network.ts

```diff
@@ -853,6 +855,47 @@ export function handleOperatorRemoved(event: OperatorRemovedEvent): void {
   }
 }
 
+export function handleOperatorWhitelistUpdated(
+  event: OperatorWhitelistUpdatedEvent
+): void {
+  let entity = new OperatorWhitelistUpdated(
+    event.transaction.hash.concatI32(event.logIndex.toI32())
+  )
+  entity.operatorId = event.params.operatorId
+  entity.whitelisted = event.params.whitelisted
+
+  entity.blockNumber = event.block.number
+  entity.blockTimestamp = event.block.timestamp
+  entity.transactionHash = event.transaction.hash
+
+  entity.save()
+
+  let whitelisted = Account.load(event.params.whitelisted)
+  if (!whitelisted){
+    log.info(`Adding new whitelisted address ${event.params.whitelisted.toHexString()} to Operator ${event.params.operatorId}, this is a new Account`, [])
+    whitelisted = new Account(event.params.whitelisted)
+    whitelisted.nonce = BigInt.zero()
+    whitelisted.save()
+  }
+  let operatorId = event.params.operatorId.toString()
+  let operator = Operator.load(operatorId) 
+  if (!operator) {
+    log.error(`Executing fees change for Operator ${event.params.operatorId}, but it does not exist on the database`, [])
+    log.error(`Could not create ${operatorId} on the database, because of missing owner, publicKey and fee information`, [])
+  }
+  else {
+    if (!operator.whitelisted) {
+      operator.whitelisted = []
+    }
+    operator.operatorId = event.params.operatorId
+    operator.whitelisted = [whitelisted.id]
```

Comment:
Need to add logic to handle `isPrivate`. I have explained in previous comments: if `event.params.whitelisted` has value, set to `true`, if not, set to `false`.